### PR TITLE
[SC_Help] adding undocumented server methods

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**
[SC_Help] adding undocumented server methods: unresponsive; sampleRate; actualSampleRate; remoteControlled;

### CODE REVIEW

File: HelpSource/Classes/Server.schelp [hunk 1]
```diff
@@ -594,6 +594,20 @@ Stops recording, closes the file, and frees the associated resources.
 discussion::
 You must call this when finished recording or the output file will be unusable. Cmd-. while recording has the same effect.
 
+method:: unresponsive
+Returns:: true/false depending if server is unresponsive
+
+method:: sampleRate
+Returns:: a variable with the current sample rate (integer)
```
Comment (vivid-synth) [inline HelpSource/Classes/Server.schelp]: It doesn't really return a variable, does it?

File: HelpSource/Classes/Server.schelp [hunk 2]
```diff
@@ -594,6 +594,20 @@ Stops recording, closes the file, and frees the associated resources.
...
+
+code::
+// returns a variable containing the sample rate (integer precision) of the specified server
```
Comment (vivid-synth) [inline HelpSource/Classes/Server.schelp]: There's no code! :)

File: HelpSource/Classes/Server.schelp [hunk 3]
```diff
@@ -594,6 +594,20 @@ Stops recording, closes the file, and frees the associated resources.
...
+
+method:: actualSampleRate
+Returns:: a variable with the current sample rate (floating point precision) of the specified server
+
+method:: remoteControlled
+Returns:: true/false stating if server is unresponsive
```
Comment (vivid-synth) [inline HelpSource/Classes/Server.schelp]: Is this really right?

**Comment by brianlheim** [comment]
Closing, see #2645.

**Pull request closed without merging**